src/colorbars.jl

```diff
@@ -31,7 +33,7 @@ Finds the limits for the colorbar by taking the "z-values" for the series and pa
 which must return the tuple `(zmin, zmax)`. The default op is the extrema of the finite
 values of the input.
 """
-function get_clims(series::Series, op = ignorenan_extrema)
+function get_clims(series::Series, op = ignorenan_extrema)::Tuple{Float64, Float64}
```

Comment:
Is there a reason to specify the outputs? Maybe just leave as Tuple{Real, Real}
What is the benefit?

Reply:
This was from a previous PR, not sure why it's being flagged as a change, as it is in master. See: https://github.com/JuliaPlots/Plots.jl/blob/master/src/colorbars.jl

Reply:
See #3838

Reply:
you didn't rebase on master

Reply:
Sure, I just thought it was supposed to show the diff vs. Master? I don't recall different behavior before...